inc/functions.php

```diff
@@ -309,7 +309,7 @@ function colormag_custom_css() {
 	}
 
 	$colormag_custom_css = get_theme_mod( 'colormag_custom_css', '' );
-	if( !empty( $colormag_custom_css ) ) {
+	if( ! empty( $colormag_custom_css ) && ! function_exists( 'wp_update_custom_css_post' ) ) {
```

Comment:
No need to check for empty because `get_theme_mod` returns boolean value if no value is set :)